src/main/java/com/minecolonies/coremod/util/RecipeHandler.java

```diff
@@ -162,7 +162,17 @@ public static void init(final boolean enableInDevelopmentFeatures, final boolean
 
         //Field
         GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(ModBlocks.blockHutField, 1),
-                                                    " Y ", "X#X", " X ", 'X', WOODEN_STICK, '#', Items.LEATHER, 'Y', Blocks.HAY_BLOCK));
+                " Y ", "X#X", " X ", 'X', WOODEN_STICK, '#', Items.LEATHER, 'Y', Blocks.HAY_BLOCK));
+
+        //Supply Items
+        GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(ModItems.supplyCamp, 1), "   ", "C C", "CCC", 'C', Blocks.CHEST));;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this empty statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AEmptyStatementUsageCheck)